### PR TITLE
1708-noteForRelations

### PR DESCRIPTION
**Pull request by TobiasNx**
I added a note field to all realtions based on the elements 760-787$$i

The Info is often mostly redundant to the relation itself.

@blackwinter can you have a look.

### CODE REVIEW

File: src/main/resources/alma/fix/relatedRessourcesAndLinks.fix [hunk 1]
```diff
@@ -40,6 +40,11 @@ set_array("supplement[]")
 do list(path:"77008", "var":"$i")
   if any_match("$i.i", "Beil.*|Supp.*")
     add_field("supplement[].$append.test","")
+    set_array("supplement[].$last.note[]")
+    do list(path:"$i.i","var":"$j")
+      copy_field("$j","supplement[].$last.note[].$append")
+    end
+    copy_field("$i.t","supplement[].$last.label")
```
Comment (blackwinter) [inline src/main/resources/alma/fix/relatedRessourcesAndLinks.fix:47]: This line is duplicated.

**Review by blackwinter — CHANGES_REQUESTED** [review]

**Review by blackwinter — APPROVED** [review]
Thanks!

**Comment by TobiasNx** [comment]
We should discuss if we want the redundant info. Beil. = supplement.

**Comment by blackwinter** [comment]
> We should discuss if we want the redundant info. Beil. = supplement.

_We_ do, yes.

**Comment by dr0i** [comment]
Is this ready to be merged @TobiasNx ? (Discussions can later be done in a (new) issue.)

**Comment by TobiasNx** [comment]
This can be merged

**Comment by dr0i** [comment]
Will be deployed next Monday.

**Comment by blackwinter** [comment]
:+1: Thanks!

**Pull request merged**